src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -120,6 +117,19 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
                 value = (theBits & ((int)flag << WellKnownAttributeDataOffset)) != 0;
                 return (theBits & ((int)flag << WellKnownAttributeCompletionFlagOffset)) != 0;
             }
+
+            public bool SetFlowAnalysisAnnotations(FlowAnalysisAnnotations value)
+            {
+                int bitsToSet = (1 << FlowAnalysisAnnotationsCompletionBit) | (((int)value & FlowAnalysisAnnotationsMask) << FlowAnalysisAnnotationsOffset);
+                return ThreadSafeFlagOperations.Set(ref _bits, bitsToSet);
```

Comment:
>bitsToSet [](start = 63, length = 9)

Consider asserting that ```bitsToSet``` do not fall out of expected bits range.

Reply:
That assert would look like the line above.

---
In reply to: [286267717](https://github.com/dotnet/roslyn/pull/35846#discussion_r286267717) [](ancestors = 286267717)

Reply:
> That assert would look like the line above.

I don't think it would.

---
In reply to: [286311129](https://github.com/dotnet/roslyn/pull/35846#discussion_r286311129) [](ancestors = 286311129,286267717)